change/@fluentui-react-components-c0066645-e657-428b-b519-715daebe27e2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31853&commitId=b56e1fd58bc44356bfb7cbd91b78ee370d5c1ba1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|6| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.badgeMask.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>